docs/concepts/what-is-a-list-filter.md

```diff
@@ -31,6 +31,20 @@ A List Filter contains [Search Attribute](/concepts/what-is-a-search-attribute)
   - **BETWEEN ... AND**
   - **IN**
   - **ORDER BY**
+  - **LIKE**
+
+- Wildcards ('\*', '%', etc.) are supported for string-type List Filter Search Attributes.
+
+  - Use wildcards with the **LIKE** operator to query possible values:
```

Comment:
```suggestion
- Wildcards ('\*', '%', etc.) are supported for string-type List Filter Search Attributes.
    Use wildcards with the **LIKE** operator to query possible values:
```

small nit - just merge this sentence with the Wildcard bullet instead of making it a sub bullet